Python/Test Data Generator/Test data generator.ipynb

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
Have a raw Python Script as well which would make it easy for us to review.

Reply:
So, should I include a file in .py format along with the Jupyter file?